wordpress-mobile/WordPress-Android
# Add analytics to track undo/redo and other missing editor actions in Aztec

This is part of the changes required to fix https://github.com/wordpress-mobile/WordPress-Android/issues/10360 - It adds the following analytics to Aztec there were missing or not correctly bumped into Tracks:

- Bold
- Italics
- Align (Left | Center | Right)
- Horizontal Rule
- Undo
- Redo

To test:
- Open Aztec
- Press the buttons mentioned above and check in LogCat that analytics are bumped
- Open the option menu, and try undo/redo. Make sure that everything is bumped correctly in analytics

Please, before merging this PR, try to merge this other PR first: https://github.com/wordpress-mobile/WordPress-Android/pull/10367 🙇 

<del>Note: There is a gutenberg-mobile PR on the way, that keeps Aztec version in par: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1289</del>
Aztec PR, already merged: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1289

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
